docs/course_materials/exercises/04_Compound_data_types/class_exercise_1.md

```diff
@@ -0,0 +1,10 @@
+### Let's Practice Together \#1 
+
+
+- Create a file named *my_list.py*
+- Create a new list with the following items: "p","k","p" ,"p","4","J","p"
+- Append "b" to the list you created
+- Remove item in index 1
+- Replace all "p" values with "o"
+- Replace "4" value with "d"
+- Replace the first item with "G"
```

Comment:
Please add the image at the end
<div style="text-align:center"><img data-natural-width="340" data-natural-height="234" style="" data-src="https://posteo.de/images/daten.png"/>